subprojects/docs/src/docs/userguide/task_configuration_avoidance.adoc

```diff
@@ -168,6 +189,21 @@ check.configure {
     dependsOn verificationTask
 }
 ----
+=====
+[.multi-language-sample]
+=====
+[source,kotlin]
+----
+val check = tasks.register("check")
+val verificationTask = tasks.register("verificationTask") {
```

Comment:
:x: can be simplified to `val verificationTask by tasks.registering {` to avoid repeating the task name